refs/ietf.json

```diff
@@ -45931,7 +45931,7 @@
     "rfc3074": {
         "rfcNumber": "RFC3074",
         "href": "https://datatracker.ietf.org/doc/html/rfc3074",
-        "title": "DHC Load Balancing Algorithm",
+        "title": "DHC [sic] Load Balancing Algorithm",
```

Comment:
I'm curious about the story here. They're consistent in that the ipv6 version carries forward this spelling